src/brogue/MainMenu.c

```diff
@@ -648,11 +648,13 @@ void scum(unsigned long startingSeed, short numberOfSeedsToScan, short scanThrou
     getAvailableFilePath(path, LAST_GAME_NAME, GAME_SUFFIX);
     strcat(path, GAME_SUFFIX);
 
-    printf("Brogue seed catalog, seeds %li to %li, through depth %i.\n\n\
+    printf("Brogue seed catalog, seeds %llu to %llu, through depth %i.\n\n\
 To play one of these seeds, press control-N from the title screen \
 and enter the seed number. Knowing which items will appear on \
 the first %i depths will, of course, make the game significantly easier.",
-            startingSeed, startingSeed + numberOfSeedsToScan - 1, scanThroughDepth, scanThroughDepth);
+            (unsigned long long)startingSeed,
+            (unsigned long long)(startingSeed + numberOfSeedsToScan - 1),
```

Comment:
Are there warnings when implicitly casting `uint64_t` to ULL?

Reply:
Yes, in fact, visible when compiling with `-Wformat`:
```
warning: format ‘%llu’ expects argument of type ‘long long unsigned int’, but argument 3 has type ‘uint64_t’ {aka ‘long unsigned int’} [-Wformat=]
```

Reply:
Ah yes, we have `-Wno-format` as that class of warnings can be so annoying, but I have no problem with correctness :)